sdk/identity/identity/CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 - Fixed browser bundling for Azure Developer CLI credential.
 
 ### Other Changes
-
+- Updated docs links.
```

Comment:
there should be a newline between the 3.3.2 other changes text and the 3.3.1 header

Reply:
`Fixed links to documentation.`

Reply:
done